src/spdx-validator.js

```diff
@@ -0,0 +1,10 @@
+const valid = require('spdx-expression-validate');
+
+const isSpdxCompliant = license => {
+  const isValid = valid(license);
+  if (!isValid) throw new Error(`Error: License ${license} is not SPDX compliant`);
```

Comment:
Could we add a link so the user can see the valid values? 

https://spdx.org/licenses/